src/resources/js/slide-toggle.js

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable no-var */
+/**
+ * Makes sure we have all the required levels on the Tribe Object
+ *
+ * @since TBD
+ *
+ * @type {PlainObject}
+ */
+tribe.events = tribe.events || {};
+
+/**
+ * Configures Slide Toggle Object on the Global Tribe variable
+ *
+ * @since TBD
+ *
+ * @type {PlainObject}
+ */
+tribe.events.slideToggle = tribe.events.slideToggle || {};
+
+/**
+ * Initializes in a Strict env the code that manages the Slide Toggle.
+ *
+ * @since TBD
+ *
+ * @param  {PlainObject} $   jQuery
+ * @param  {PlainObject} obj tribe.events.slideToggle
+ *
+ * @return {void}
+ */
+(function ( $, obj ) {
```

Comment:
I don't have the full context here, but a couple thoughts:
1. We have an `accordion.js` in TEC, I wonder if we can copy over to common and use it there rather than creating yet another JS file. It's a generic accordion functionality that sets up an accordion based on the aria attributes. We can think about a replacement plan for the TEC accordion JS file later, but that functionality should have belonged to common anyway.
2. Curious on the name slide toggle, rather than accordion or something else, was there a specific reason for this name?

Reply:
Hi @paulmskim 
Thanks, I missed that script. So I am changing to use the accordion.js in tec. We had already used it in VE, but then that part had been removed. 

I named it after the jquery function, but it is gone now and only the switch will go to common.